### ISSUE TITLE
`wazuh-syscheckd` not found syscheck fim strees file

### ISSUE THREAD

**Issue by fcaffieri** [wazuh/wazuh#18175]
### Description

Within the footprint tests carried out for Wazuh 4.5.0 version, we found the following error within the module `wazuh-syscheckd`:

Test:
- Version: 4.5.0
  - Issue: https://github.com/wazuh/wazuh/issues/18102
  - Jenkins build: https://ci.wazuh.info/job/Test_stress/4200/
  - Target: Manager 4.5.0
  - Repository: packages-dev.wazuh.com
  - Bucket: pre-release

Log errors:

```
wazuh-syscheckd: ERROR: (6704): Couldn't get path of '/tmp/syscheck_test/files/fimStress.890531'
```

**Comment by mauromalara** [comment]
@fcaffieri I think this is duplicated (https://github.com/wazuh/wazuh-jenkins/issues/5287)

**Comment by vikman90** [comment]
@fcaffieri @mauromalara I don't think so. https://github.com/wazuh/wazuh-jenkins/issues/5287 reports that the agent is preventing 3rd party applications (the stress test script) from removing files.

On the other hand, I'm pretty sure that this happens because the file has been deleted while the manager detected that it was not synced and requested an update.

This is a low-impact bug.

Best!
